systemd-boot/config

```diff
@@ -10,4 +10,11 @@
 # include common functions
 . "$PBL_INCLUDE/library"
 
-echo "nothing to do, really"
+if [ -x /usr/bin/sdbootutil ] ; then
+  ( set -x ; sdbootutil update-all-entries ) || err=1
+elif [ -x /usr/bin/kernel-install ] ; then
```

Comment:
I would not use this yet?

sdbootutil is a replacement for opensuse for kernel-install, so if there is a situation in openSUSE when grub2-bls or systemd-boot is used and sdbootutil is not present, calling kernel-install can damage the system

Reply:
This is just for consistency with the other files: `add-kernel` also uses `kernel-install` already

Reply:
That was a mistake : (